lib/wasi/src/state/builder.rs

```diff
@@ -76,7 +80,7 @@ pub enum WasiStateCreationError {
     #[error("wasi filesystem setup error: `{0}`")]
     WasiFsSetupError(String),
     #[error(transparent)]
-    WasiFsError(WasiFsError),
+    FsError(FsError),
```

Comment:
The other variants are named `WasiFs*`, so let's keep `WasiFsError` here?

Reply:
In this case I believe it will not be a WasiFsError but a FileSystem error. I agree we need some consistency on the error names though